release-notes/docs/3.7.0.md

```diff
@@ -0,0 +1,180 @@
+# 3.7.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Configurable hierarchies](#configurable-hierarchies)
+- [Complete reference configuration](#complete-reference-configuration)
+- [Optimised form HTML generation](#optimised-form-html-generation)
+- [Purging server-side](#purging-server-side)
+- [Warn when replicating too many docs](#warn-when-replicating-too-many-docs)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+A new OS level dependency (`xsltproc`) is required for generating XFORM `html` on the server. 
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## Configurable hierarchies
+
+It is now possible to configure contact hierarchies.
+This allows for better tailored solutions for new projects and opens up new configuration 
+possibilities: for example having multiple person types or unbalanced place trees. 
```

Comment:
I would highlight that you could have 1 or more places as well since that is significant for many deployments.

```suggestion
possibilities: for example having any number of place levels, multiple person types, or unbalanced place trees. 
```